build_rules.gradle

```diff
@@ -70,6 +70,12 @@ repositories {
 // Apply a plugin which enables configuring projects imported into Intellij.
 apply plugin: "idea"
 
+idea {
+  module {
+    inheritOutputDirs = true;
```

Comment:
Can you add a comment as to why this is being set instead of taking the default, specifically what does it fix.